grub2_theme_preview/__main__.py

```diff
@@ -144,9 +144,9 @@ def _make_grub_cfg_load_our_theme(grub_cfg_content, source_type, resolution_or_n
     epilog_chunks = [
         # Ensure that we always have one or more menu entries
         '',
-        'submenu \'Reboot / Shutdown\' {',
-        '    menuentry Reboot { reboot }',
-        '    menuentry Shutdown { halt }',
+        'submenu \'Reboot / Shutdown\' --class shutdown {',
+        '    menuentry Reboot --class restart { reboot }',
+        '    menuentry Shutdown --class shutdown { halt }',
```

Comment:
If this has practical value, please move it to a dedicated pull request and elaborate on it's practical effect. Thank you!